test/unit/search/query_components/core_query_test.rb

```diff
@@ -20,27 +20,13 @@ class CoreQueryTest < ShouldaUnitTestCase
     end
   end
 
-  context "when ab testing minimum should match" do
-    should "use the default value when no variant is passed in" do
-      builder = QueryComponents::CoreQuery.new(search_query_params)
-
-      query = builder.payload
-      assert_match(/"2<2 3<3 7<50%"/, query[:bool][:must].to_s)
-    end
-
-    should "use variant b when it is passed in" do
+  context "minimum should match" do
+    should "is part of the should clause" do
```

Comment:
Super nitpicky: the wording "should is" is a bit odd.

I wonder how we would word this if we were writing it from scratch rather than as part of an A/B test. Maybe something like:

```
context "the search query" do
  should "down-weight results which match fewer words in the search term" do
```